lib/callbacks/onegini_custom_registration_callback.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/services.dart';
+import 'package:onegini/constants/constants.dart';
+
+import '../onegini.dart';
+
+/// A callback to respond to the init and finish of customRegistrationActions.
```

Comment:
Class name should be self-explanatory, there should be no need for a comment.

Reply:
done